hail/python/hail/docs/change_log.md

```diff
@@ -22,6 +22,22 @@ Please note that **forward compatibility should not be expected, especially
 relating to file formats**: this means that it may not be possible to use
 an earlier version of Hail to read files written in a later version.
 
+## Version 0.2.50
+
+Released 2020-07-22
+
+### Bug fixes
+
+- (hail#9114) CHANGELOG: Fixed crash when using repeated calls to `hl.filter_intervals`.
+
+### New features
+
+- (hail#9101) Add `hl.nd.concat` to concatenate ndarrays.
+- (hail#9105) Add `hl.nd.eye` to create identity matrix ndarrays.
+- (hail#9093) Add `hl.nd.inv` to invert ndarrays.
+
```

Comment:
Things that are missing:

- Alex also added `hl.nd.hstack`, `hl.nd.vstack`, and `hl.nd.identity`, but these just call out to the 3 mentioned above.
- My fault for apparently not having a CHANGELOG, but #9063 introduced `tree_matmul` on BlockMatrix